src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.controllers;
+
+import io.swagger.annotations.ApiOperation;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.reports.EnvelopeCountSummaryReportResponse;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.reports.EnvelopeCountSummary;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.reports.ReportsService;
+
+import java.time.LocalDate;
+
+import static org.springframework.format.annotation.DateTimeFormat.ISO.DATE;
+
+@RestController
+@RequestMapping(path = "/reports", produces = MediaType.APPLICATION_JSON_VALUE)
+public class ReportsController {
+
+    private final ReportsService reportsService;
+
+    public ReportsController(ReportsService reportsService) {
+        this.reportsService = reportsService;
+    }
+
+    @GetMapping(path = "/count-summary")
+    @ApiOperation("Retrieves envelope count summary report")
+    public EnvelopeCountSummaryReportResponse getCountSummary(
+        @RequestParam(name = "date") @DateTimeFormat(iso = DATE) LocalDate date
```

Comment:
How about path variable?

Reply:
should we accept start date and end date rather than just date. This way we can get reports for a particular month/week if required. Not sure if it required now but something may be nice to have

Reply:
Or we might need to have daily counts for a specific period. I don't know. Maybe we should stick to the simplest version and once something extended is required, we can adjust.

Reply:
I went with this as it's both simple and allows extension with other filtering in the future (for example `endDate` as @nitinprabhu suggested, but also `jurisdiction` etc)